nuxt.config.ts

```
@@ -1,8 +1,11 @@
+import process from 'node:process'
+
 const siteUrl = 'https://xanzhu.com'
 const cdnUrl = 'https://cdn.xanzhu.com'
+const isDev = process.env.NODE_ENV === 'development'
 
 export default defineNuxtConfig({
```

ESLint

[error] 'defineNuxtConfig' is not defined.

(no-undef)